File: truyenvkl/toc.lua
local url = ...
local list = {}
local doc = http:get(url):html()
if doc ~= nil then
    local el = doc:select(".listchap li a")
    for i = 0, el:size() - 1 do
        local e = el:get(i)
        local chap = {}
        chap["name"] = e:text()
        chap["url"] = e:attr("href")
        chap["host"] = "https://truyenvkl.com"
        table.insert(list, chap)
    end
    return response:success(list)
end
return nil

File: saytruyen/toc.js
function execute(url) {
    var doc = Http.get(url).html();

    var el = doc.select("#list_container").select("a");
    const data = [];
    for (var i = el.size() - 1; i >= 0; i--) {
        var e = el.get(i);
        data.push({
            name: e.text(),
            url: e.attr("href"),
            host: "https://saytruyen.com"
        })
    }

    return Response.success(data);
}